iceoryx_posh/include/iceoryx_posh/runtime/posh_runtime.hpp

```diff
@@ -109,15 +109,16 @@ class PoshRuntime
     SubscriberPortUserType::MemberType_t*
     getMiddlewareSubscriber(const capro::ServiceDescription& service,
                             const popo::SubscriberOptions& subscriberOptions = popo::SubscriberOptions(),
-                            const NodeName_t& nodeName = "",
+                            const NodeName_t& nodeName = NodeName_t(cxx::TruncateToCapacity, ""),
                             const PortConfigInfo& portConfigInfo = PortConfigInfo()) noexcept;
 
     /// @brief request the RouDi daemon to create an interface port
     /// @param[in] interface interface to create
     /// @param[in] nodeName name of the node where the interface should belong to
     /// @return pointer to a created interface port data
     popo::InterfacePortData* getMiddlewareInterface(const capro::Interfaces interface,
-                                                    const NodeName_t& nodeName = "") noexcept;
+                                                    const NodeName_t& nodeName = NodeName_t(cxx::TruncateToCapacity,
+                                                                                            "default")) noexcept;
```

Comment:
again, why `"default"`? It's not changed for `getMiddlewarePublisher` or `getMiddlewareSubscriber`